modules/ROOT/pages/introduction-to-tiny-spellchecker.adoc

```diff
@@ -211,6 +211,56 @@ tinymce.init({
 });
 ----
 
+=== SpellcheckerUpdated Event
+
+NOTE: Requires 6.5.
+
+This event is fired when the editor is spellchecked either by opening the dialog or by executing the `+mceSpellcheckUpdate+` command.
```

Comment:
```suggestion
include::partial$misc/admon-requires-6.5v.adoc[]

This event is fired when the editor is spellchecked: either by opening the Spell Checker Pro dialog, or by executing the `+mceSpellcheckUpdate+` command.
```
1. In anticipation of [PR2761](https://github.com/tinymce/tinymce-docs/pull/2761) being approved and merged.
2. copy-edits and a qualifier for specificity’s sake.

Reply:
I have some more changes ready here so I'll include this once that PR is merged